libs/membership/website/src/lib/invoice-list/invoice-list-item.tsx

```diff
@@ -152,10 +152,17 @@ export function InvoiceListItem({
         )}
 
         {/* @TODO: Remove when all 'payrexx subscriptions' subscriptions have been migrated  */}
+        {/* @TODO: Make href used in <Link> component customizable if necessary. You may want to make the custom Bajour filters default filters. */}
         {showPayrexxSubscriptionWarning && (
-          <Alert severity="warning">
-            Wir haben vor einiger Zeit das Membersystem umgestellt. Du verfügst noch über eine alte
-            Membership, die automatisch abgebucht wird.
+          <Alert
+            severity="warning"
+            action={
+              <Link
+                href={`/mitmachen?memberPlanBySlug=${subscription?.memberPlan.slug}&upsell=true&deactivateSubscriptionId=${subscription?.id}`}>
```

Comment:
Read comment above in the code. I know this is a Bajour specific url for now. We can change it if needed. Params wouldn't hurt anyway. Thinking about moving those filter params to standard functionality of Website Builder. Use cases would be the same for other media.